src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -104,6 +109,7 @@
     <Compile Include="WeakEventHandlerManager.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\ruleLookup.json" />
```

Comment:
Eventually we should just include this file in the package. I'll make that change after you commit.